src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -169,7 +170,32 @@
     <_MakePriConfigFile>$(_MakePriHelpersDir)/ModifiedConfigFile.xml</_MakePriConfigFile>
     <_ReswListFile>$(_MakePriHelpersDir)/reswlist.RESFILES</_ReswListFile>
     <_PriListFile>$(_MakePriHelpersDir)/prilist.RESFILES</_PriListFile>
+    <_ExternalReswOutputPath>$(ResourcesFolderPath)/external/</_ExternalReswOutputPath>
   </PropertyGroup>
+  
+  <ItemGroup Condition="'$(BuildingUAPVertical)' == 'true'">
+    <_AllRuntimeDllFiles Include="$(RuntimePath)\*.dll" />
+    <_ExternalReswFiles Include="$(_ExternalReswOutputPath)*.resw" />
+    
+    <!-- The first time the CreateReswFilesForExternalDependencies target is executed the itemgroup _ExternalReswFiles will be empty
+    and that will avoid the target from executing, so we add a dummy file if they are empty so that the target is executed the first time. -->
+    <_ExternalReswFiles Condition="@(_ExternalReswFiles) == ''" Include="$(_ExternalReswOutputPath)dummy.resw" />
```

Comment:
We have no way to know what the resw file is going to be as that information comes from the dll's metadata, it can be a custom name, i.e we add an embedded resource for testing purposes that it's resources file is called SantiResources.resx and I can override the resource manifest name to be `Santi.This.IsMyResourceManager` in the csproj like in [here](https://github.com/dotnet/corefx/blob/master/src/System.Composition.Hosting/src/System.Composition.Hosting.csproj#L72) and that would mean that the resw file needs to be called `Santi.This.IsMyResourceManager.resw`

Since that is embedded into the dll's metadata there is no way to know what resw files the task will output and that is why the output resw files I create them [here](https://github.com/dotnet/buildtools/pull/1630/files#diff-4dba46151dfcc864a9186f4ae3de64c1R56) which comes from the ResourceName. That is why in the beginning I made the Outputs to be the common pri file, because that one is static, and if it is newer than the dlls then we don't need to update it.